docs/templates/epacems_child.rst.jinja

```diff
@@ -56,18 +54,17 @@ A plain English explanation of the requirements of Part 75 is available in secti
 
 {% block original_data %}
 EPA CAMD publishes the CEMS data in an online `data portal <https://campd.epa.gov/>`__.
-The files are available in a prepackaged format, accessible via a `user interface <https://campd.epa.gov/data/custom-data-download>`__
-or `FTP site <https://gaftp.epa.gov/DMDnLoad/>`__ with each downloadable zip file
-encompassing a year of data.
+The files are available in a prepackaged format, accessible via a `user interface
+<https://campd.epa.gov/data/custom-data-download>`__.
```

Comment:
Worth mentioning the API here, which is how we actually access the data?
https://www.epa.gov/power-sector/cam-api-portal

Reply:
Added a link to the CAM API.